File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Integer between 0 and 23: ");
    }
    while (height < 0 || height > 23);
    for (int i = 0; i < height; i++)
    {
        // prints spaces for left portion
        for (int spaces = height - 1; spaces > i; spaces--)
        {
            printf(" ");
        }
        // prints # for left portion
        for (int x_s = -1; x_s < i; x_s++)
        {
            printf("#");
        }
        // prints double gap
        printf("  ");
        // prints # for right portion
        for (int x_s = -1; x_s < i; x_s++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset5/mario_more/mario.py
import cs50

while(True):
    height = cs50.get_int("Integer between 0 and 23: ")
    if height >= 0 and height <= 23:
        break

t = height
i = 1
for x in range(0, height):
    # print spaces for left
    t = t - 1
    print(' ' * t, end="")

    # print # for left portion
    print('#' * i, end="")

    # print double gap
    print('  ', end="")

    # print # for right portion
    print('#' * i)

    # change # for next row
    i = i + 1
